sdk/go/common/resource/plugin/analyzer_plugin.go

```diff
@@ -103,13 +103,13 @@ func NewPolicyAnalyzer(
 	// Load the policy-booting analyzer plugin (i.e., `pulumi-analyzer-${policyAnalyzerName}`).
 	pluginPath, err := workspace.GetPluginPath(ctx.Diag,
 		workspace.AnalyzerPlugin, policyAnalyzerName, nil, host.GetProjectPlugins())
-	if err != nil {
-		return nil, rpcerror.Convert(err)
-	} else if pluginPath == "" {
+	if _, ok := err.(*workspace.MissingError); ok {
```

Comment:
use errors.Is, not just a cast check.

Reply:
It's suggested to do `errors.As()`

https://github.com/golang/go/wiki/ErrorValueFAQ#how-should-i-change-my-error-handling-code-to-work-with-the-new-features

MissingError is created with NewMissingError and has arguments so testing for equality would be hard to do or fragile.